### DISCUSSION TITLE
Add TypeScript support to extensions

### DISCUSSION THREAD

**Discussion by apstanisic** [directus/directus#3058] [Feature Requests]
It would be a lot easier to write API extensions if we had types for services

**Comment by nickluger** [comment]
See workaround here: https://github.com/directus/directus/discussions/3025#discussioncomment-581252

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
